books/kestrel/crypto/r1cs/sparse/p1cs.lisp

```diff
@@ -195,8 +202,8 @@
            (stringp (cadr (assoc-equal key alist))))
   :hints (("Goal" :in-theory (enable assoc-equal))))
 
-;; This shows there are no collisions:
-;; (assert-equal (len *nums-to-strings*) (len (remove-duplicates (strip-cars *nums-to-strings*))))
+;;This shows there are no collisions:
+;;(assert-equal (len *nums-to-strings*) (len (remove-duplicates (strip-cars *nums-to-strings*))))
```

Comment:
This comment should be updated to reflect the other changes.   Run the assert-equal to make sure it was updated correctly.